code/modules/vending/wardrobes.dm

```diff
@@ -88,6 +101,9 @@
 					/obj/item/clothing/suit/hooded/wintercoat/engineering/atmos = 3,
 					/obj/item/clothing/under/rank/atmospheric_technician = 3,
 					/obj/item/clothing/shoes/sneakers/black = 3)
+	contraband = list(/obj/item/extinguisher = 1,
+					  /obj/item/extinguisher/mini = 2)
+	premium = list(/obj/item/extinguisher/advanced = 2)
```

Comment:
More gear that doesn't make sense in the wardrobe. Base Extinguishers as contraband?? Why even bother??? Move Advanced to contraband and maybe it's worth actually hacking it but why is it in a WARDROBE!!!